x-pack/plugins/maps/common/elasticsearch_util/spatial_filter_utils.ts

```diff
@@ -140,7 +140,6 @@ export function createSpatialFilterWithGeometry({
 
   // Currently no way to create an object with exclude property from index signature
```

Comment:
Can you remove comments since TS error no longer occurs and they describe the error.